### ISSUE TITLE
Question about the status

### ISSUE THREAD

**Issue by Narrat** [milkypostman/caw#3]
Hi

I just wanted to know if this project is somewhat active.
There are some serious problems that prevents caw from running on recent linux machines (battery and volume widget not working, xcb.XA_* needs to be renamed and some more).
I could provide information about problems or PRs but if this is considered dead this would be somewhat useless.

Best regards

**Comment by milkypostman** [comment]
Yeah I don't really use it at all. I have accepted patches in the past and
would be happy to have someone take over commits if they still used it!

**Comment by Narrat** [comment]
Dunno about how many use it/would use it.
But imo it would be nice to have still the choice. Many panel applications are somewhat on hold.
And what I could see about caw made a good impression, if the problems can be solved.
If I can help in some way, why not

**Comment by milkypostman** [comment]
Well, I'm happy to accept patches but don't have enough time, since I
don't use it anymore, to really figure out what needs to be done.

Wish I could be of more help; too many other things to work on.
